src/compile/mark/mark.ts

```diff
@@ -83,6 +85,170 @@ function parsePathMark(model: UnitModel) {
   }
 }
 
+const STACK_GROUP_PREFIX = 'stack_group_';
+
+/**
+ * We need to put stacked bars into groups in order to enable cornerRadius for stacks.
+ * If stack is used and the model doesn't have size encoding, we put the mark into groups,
+ * and apply cornerRadius properties at the group.
+ */
+function getStackGroups(model: UnitModel) {
+  // Don't use nested groups when cornerRadius is not specified
+  const hasCornerRadius = VG_CORNERRADIUS_CHANNELS.some(
+    prop => model.markDef[prop] != null || getMarkConfig(prop, model.markDef, model.config) != null
```

Comment:
We use strict comparison (`!==`) in general.  

Maybe we should turn this on for linting in a separate PR.

Reply:
`!=` is actually checking for both `null` and `undefined`, changing to `!==` is probably going to break. Do we have a helper function for this?

Reply:
oh I'm wrong, `!==` is the right thing here.